src/pages/in/invoice-software.js

```
@@ -1,9 +1,1158 @@
+              >
+                <div className='col-md-12 col-lg-6'>
+                  <div className='features__description_container__content'>
+                    <h2 className='col-primary c-fw-600 mb-3' role="heading" aria-level="3">
+                      Run your business with automation!
+                    </h2>
+                    <p>
+                      From setting payment reminders to sending{" "}
+                      <a
+                        href='https://giddh.com/blog/benefits-of-online-invoicing-for-small-business/'
+                        target='_blank'
+                        className='text-decoration-underline'
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)